lib/resty/core/phase.lua

```diff
@@ -0,0 +1,55 @@
+local ffi = require 'ffi'
+local C = ffi.C
+local base = require "resty.core.base"
+local FFI_ERROR = base.FFI_ERROR
+local tostring = tostring
+
+ffi.cdef[[
+int ngx_http_lua_ffi_get_phase(ngx_http_request_t *r, char **err)
+]]
+
+
+local _M = {
+    _VERSION = base.version
```

Comment:
It seems other modules in this library export the version number in a `version` field (non-capitalized, no `_` prefix). They also avoid creating a `_M` module table if not necessary. Shouldn't we try staying consistent? 

Example: https://github.com/openresty/lua-resty-core/blob/master/lib/resty/core/time.lua